server/app.js

```diff
@@ -12,11 +12,12 @@ var householdRouter = require('./routes/householdRouter');
 var studentRouter = require('./routes/studentRouter');
 var activityRouter = require('./routes/activityRouter');
 var schoolInfoRouter = require('./routes/schoolInfoRouter');
+var paymentRouter = require('./routes/paymentRouter')
 var emailRouter = require('./routes/emailRouter')
 
 var app = express();
 
-// our dev server and client run on 2 different ports 
+// our dev server and client run on 2 different ports
 // so we need to whitelist the client address
 if (app.get('env') === 'development') {
   const whitelist = ['http://localhost:3000', 'https://bright-futures-academy-dev.herokuapp.com/']
```

Comment:
If you want to check staging, add "bright-futures-academy-staging.herokuapp.com" here